src/mscorlib/shared/Interop/Windows/Interop.Libraries.cs

```diff
@@ -11,5 +11,6 @@ internal static partial class Libraries
         internal const string Kernel32 = "kernel32.dll";
         internal const string OleAut32 = "oleaut32.dll";
         internal const string User32 = "user32.dll";
+        internal const string FileApiInterop = "FileApiInterop.dll";
```

Comment:
The CoreFX part of the change does not have it as constant: https://github.com/dotnet/corefx/pull/23316/files#diff-b609925cb27caaff205ea21bde48c3f2R15 . I think it would be nice to not have it as constant here either.